deploy/operator/utils.sh

```diff
@@ -62,6 +62,14 @@ function wait_for_condition() {
     for i in {1..40}; do
         oc get ${object} --selector="${selector}" --namespace=${namespace} |& grep -ivE "(no resources found|not found)" && break || sleep 10
     done
+    if [[ "${object}" == "crd/agentserviceconfigs.agent-install.openshift.io" ]]; then
+        echo "## Install plan is"
+        oc get InstallPlan -n assisted-installer -o yaml
+
+        echo "## Catalogsource is"
+        oc get catalogsource assisted-service-operator-catalog -n openshift-marketplace -o yaml
+        sleep 60m
```

Comment:
To merge we will need to remove the sleep here because it will cause timeout failure. In the past for live debug I put the sleep at the end of the step in the step registry.

But let's add these debug statements in I think it will help long term.